posix.mak

```diff
@@ -653,16 +653,19 @@ test: $(ASSERT_WRITELN_BIN)_test all
 ################################################################################
 # Changelog generation
 ################################################################################
+CHANGELOG_FILES=$(wildcard $(DMD_DIR)/changelog/*.dd) \
+				$(wildcard $(DRUNTIME_DIR)/changelog/*.dd) \
+				$(wildcard $(PHOBOS_DIR)/changelog/*.dd)
```

Comment:
Should we add `tools` here too? It has only been used once so far.

Reply:
Yes, ideally all repos that can have changelog entries - `installer` also looks like a good candidate. Though, I'm not sure about dub.